__tests__/cli.ts

```diff
@@ -0,0 +1,76 @@
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import childProcess from 'child_process';
+
+const mkdir = util.promisify(fs.mkdir);
+const rmdir = util.promisify(fs.rmdir);
+const writeFile = util.promisify(fs.writeFile);
```

Comment:
This project already has a [util wrapper around `fs.writeFile`](https://github.com/OpenSourceRaidGuild/unimported/blob/master/src/fs.ts#L29) that I contemplated using insead of this, but I ultimately chose not to.  I prefer not tying some hacky test usage to the production implementation, but I'm not too hung up about it if others prefer it.

Reply:
Isn't there an `fs/promise`? Would it do the same thing?

Reply:
google says yes, I'll update it tomorrow.

Reply:
@JacobMGEvans turns out this changed between node versions:

<= 12: `import { promises as fs } from 'fs'`
> 12: `import fs from 'fs/promises'`

We've been discussing running the tests in a matrix of node versions in discord and `promisify` is consistent for all stable and LTS versions so I'm suggesting we should keep it as is.